docs/python/examples.ipynb

```diff
@@ -751,7 +734,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The highlighted regions show where a pump is active.\n"
+    "We see that in Januari the level of the basin is too high and thus water is pumped out until the maximum level of the desried range is reached. Then until May water flows out of the basin freely trough the tabulated rating curve until the minimum leavel is reached. From \n",
```

Comment:
Several typos here (desried, leavel, trough)

Reply:
Ah, yes, I typed this and then lost it, then typed it again in about 30 seconds in a fit of rage 😋